cfg/ECAL_H4_July2021/ECAL_H4_base.cfg

```diff
@@ -41,6 +41,7 @@ maxClusterSize 4
 #---VFE config
 <DigiReco>
 pluginType DigitizerReco
+litedtu 1
```

Comment:
I don't think there is any need to have this here. The WFClass type can already be specified using type inside a channel block like in this [cfg].

[cfg]:https://github.com/simonepigazzini/H4Analysis/blob/master/cfg/ECAL_H4_October2018/ECAL_H4_Oct2018_base_160MHz.cfg#L252

Reply:
right, it's cleaner like that